app/src/main/kotlin/com/wire/android/ui/edit/CopyItemMenuOption.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.edit
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.wire.android.R
+import com.wire.android.ui.common.bottomsheet.MenuBottomSheetItem
+import com.wire.android.ui.common.bottomsheet.MenuItemIcon
+
+@Composable
+fun CopyItemMenuOption(onCopyItemClick: () -> Unit) {
```

Comment:
Maybe a good candidate to move to commons ? 🤔 
But no fixed personal preference on this :D

Reply:
Nice idea but it would be better to move all menu options (9 elements), one of them is reactions so also whole emoji picker thing with its bottom sheet. Also, move or copy all related resources and strings - what's the approach to that? I think we should move all of them to common-ui module and use it from there, but then we have at least two places to look at and maintain. 🤔 
Maybe it would be better to do this as a separate PR 😅